.vscode/extensions.json

```diff
@@ -4,7 +4,6 @@
 		"platymuus.dm-langclient",
 		"EditorConfig.EditorConfig",
 		"dbaeumer.vscode-eslint",
-		"coenraads.bracket-pair-colorizer-2",
```

Comment:
If you're modifying extensions please also add `usernamehw.errorlens` 

![image](https://user-images.githubusercontent.com/25063394/164053270-53549520-2ab3-492c-a6b1-68c047ac44d8.png)

It is amazing